docs/minikube.md

```diff
@@ -0,0 +1,74 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
+
+### Setup the certificate
```

Comment:
There are still other mention of wildcard domain requirements for Draft.

* https://github.com/Azure/draft/blob/master/docs/ingress.md#point-a-wildcard-domain
* https://github.com/Azure/draft/blob/master/docs/install.md#dependencies

Is it still valid?